dicom-mr-classifier.py

```diff
@@ -593,20 +593,20 @@ def dicom_classify(zip_file_path, outbase, timezone, config=None):
 
     # If no pixel data present, make classification intent "Non-Image"
     if not hasattr(dcm, "PixelData"):
-        nonimage_intent = ["Non-Image"]
+        non_image_intent = 'Non-Image'
         # If classification is a dict, update dict with intent
         if isinstance(dicom_file["classification"], dict):
             classification  = dicom_file['classification']
             # If intent not present, add it with nonimage_intent
             if 'Intent' not in classification:
```

Comment:
Does this need to be `classification.keys()`?

Reply:
It shouldn't, also enabled gear-toolkit metadata validation, with successful run here: https://ga.ce.flywheel.io/#/projects/602da1eb17d9612818a29428/sessions/611e8483b6bb3fccea3f29de?tab=provenance